packages/itwinui-css/package.json

```diff
@@ -35,17 +35,22 @@
     "bentley systems",
     "itwin"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "stylelint": "^15.10.1",
+    "stylelint-csstree-validator": "^3.0.0",
+    "stylelint-scss": "^5.0.1",
+    "stylelint-use-logical": "^2.1.0"
```

Comment:
these should all go under devDependencies

Reply:
done